packages/clerk-js/src/core/clerk.ts

```diff
@@ -288,6 +292,13 @@ export class Clerk implements ClerkInterface {
     return this.#options.standardBrowser || false;
   }
 
+  get __experimental_commerce(): __experimental_CommerceNamespace {
+    if (!Clerk._commerce) {
+      Clerk._commerce = new __experimental_Commerce();
+    }
+    return Clerk._commerce;
+  }
```

Comment:
I think we'll only want to initialize this if the experimental option is enabled, so something like this:

```suggestion
  get __experimental_commerce(): __experimental_CommerceNamespace {
    if (!this.#options.experimental.commerce) {
      throw new Error('Clerk: commerce functionality is currently in an experimental state. To enable, pass `experimental.commerce = true`.')
    }
    
    if (!Clerk._commerce) {
      Clerk._commerce = new __experimental_Commerce();
    }
    return Clerk._commerce;
  }
```